tools/ssh/README.md

```diff
@@ -34,7 +34,8 @@ Requires [comma SIM with comma prime](https://comma.ai/shop) activated with comm
 
 ## Recommended .ssh/config
 
-With the below ssh configuration, you can type `ssh comma-{dongleid}` to connect to your device through `ssh.comma.ai`. For example, `ssh comma-ffffffffffffffff`.
+With the below SSH configuration, you can type `ssh comma-{dongleid}` to connect to your device through `ssh.comma.ai`.  
```

Comment:
```suggestion
With the below SSH configuration, you can type `ssh comma-{dongleid}` to connect to your device through `ssh.comma.ai`.<br />
```

Reply:
> 

I really prefer the simplicity of the double spaces, but understand whitespace is controversial.  That said, it appears the extra space and forward slash you suggested are also unnecessary...
https://www.markdownguide.org/basic-syntax/